### PR TITLE
deflake the test for checking that we don't inline gzipped resources

### PR DESCRIPTION
**Pull request by oschaaf**

### CODE REVIEW

**Comment by oschaaf** [comment]
(this needs an amendment, tests failed)

File: pagespeed/automatic/system_tests/inliners.sh [hunk 1]
```diff
@@ -77,7 +77,7 @@ start_test inlining gzip-encoded resources
 # see two .pagespeed. resources, then verify that we see the debug comments we
```
Comment (morlovich) [inline pagespeed/automatic/system_tests/inliners.sh]: Comment needs updating?

Comment (oschaaf) [reply pagespeed/automatic/system_tests/inliners.sh]: re: "Comment needs updating?" 
done

**Comment by oschaaf** [comment]
Hmm, interesting test failure at  https://github.com/pagespeed/mod_pagespeed/blob/b6c29c5dcc78984ae1ac9811edcdcd95a4a8a380/pagespeed/system/tcp_server_thread_for_testing.cc#L138

(probably not related to this change)

```
[ RUN      ] AprMemCacheTest.HangingMultigetTest
[0706/141944:FATAL:tcp_server_thread_for_testing.cc(138)] Check failed: status == 0 (98 vs. 0)CreateAndBindSocket apr_socket_bind
out/Debug/pagespeed_automatic_test() [0x10fe700]
out/Debug/pagespeed_automatic_test() [0x10ffad1]
out/Debug/pagespeed_automatic_test() [0xf6752f]
out/Debug/pagespeed_automatic_test() [0xf677f5]
out/Debug/pagespeed_automatic_test() [0xf66f4d]
out/Debug/pagespeed_automatic_test() [0xf60de1]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x7e9a) [0x7f3f8c76fe9a]
/lib/x86_64-linux-gnu/libc.so.6(clone+0x6d) [0x7f3f8c49d2ed]

/home/travis/build/pagespeed/mod_pagespeed/install/run_program_with_memcached.sh: line 56:  6163 Aborted                 (core dumped) out/Debug/pagespeed_automatic_test
Killing memcached on port 11480 running with pid=5842

End of log/unit_test.log:
```
I think that status 98 means "Address already in use", but I'm not sure.

**Comment by morlovich** [comment]
Yeah, that failure looks like it had trouble grabbing a port for a test server for some reason.

This change:
LGTM

**Pull request merged**